src/main/java/org/jabref/gui/fieldeditors/FileListEditorTransferHandler.java

```diff
@@ -82,21 +82,21 @@ public boolean importData(JComponent comp, Transferable t) {
             List<Path> files = new ArrayList<>();
             // This flavor is used for dragged file links in Windows:
             if (t.isDataFlavorSupported(DataFlavor.javaFileListFlavor)) {
+                // javaFileListFlavor returns a list of java.io.File (as the string *File* in File indicates) and not a list of java.nio.file
+                // There is no DataFlavor.javaPathListFlavor, so we have to deal with java.io.File
                 @SuppressWarnings("unchecked")
-                List<Path> transferedFiles = (List<Path>) t.getTransferData(DataFlavor.javaFileListFlavor);
-                files.addAll(transferedFiles);
-            }
-
-            if (t.isDataFlavorSupported(urlFlavor)) {
+                List<File> transferedFiles = (List<File>) t.getTransferData(DataFlavor.javaFileListFlavor);
```

Comment:
Just a nitpick, but since you're already at it could you please change this variables name from `t` to someting more meaningful?

Otherwise the code looks fine.